libs/components/core/src/lib/modules/dock/dock-dom-adapter.service.ts

```diff
@@ -88,7 +88,7 @@ export class SkyDockDomAdapterService implements OnDestroy {
     // Create a style element to avoid overwriting any existing inline body styles.
     const styleElement = this.#renderer.createElement('style');
     const textNode = this.#renderer.createText(
-      `body { margin-bottom: ${dockHeight}px; }`,
+      `body { margin-bottom: ${dockHeight}px; --sky-dock-height: ${dockHeight}px; }`,
```

Comment:
Crux of the fix is to add a custom property to add when `<body>` `margin-bottom` is set elsewhere.